shlr/tcc/tccgen.c

```diff
@@ -755,17 +756,16 @@ static void type_to_str(char *buf, int buf_size, CType *type, const char *varstr
 	case VT_STRUCT:
 	case VT_UNION:
 		if (bt == VT_STRUCT) {
-			tstr = "struct ";
+			tstr = "struct";
 		} else if (bt == VT_UNION) {
-			tstr = "union ";
+			tstr = "union";
 		} else {
-			tstr = "enum ";
+			tstr = "enum";
 		}
```

Comment:
just use the fallthrough here, and set tstr on each of these, it's weirdly done in this case with the if when you already know the type.

like 

```
case VT_ENUM:
  tstr = "enum";
  // allow fallthrough
case VT_STRUCT:
  tstr = "struct";
  // allow fallthrough
```

also maybe there should be constants for these strings?